frontend/src/Admin/components/ShowSubmissionDetails.js

```diff
@@ -154,6 +147,16 @@ const ShowSubmissionDetails = ({ submission }) => (
         <dd>{submission.forSale ? 'Yes' : 'No'}</dd>
         <dt>More Copies?</dt>
         <dd>{submission.moreCopies ? 'Yes' : 'No'}</dd>
+        {submission.entryType === PHOTO ? (
+          <div className='text-center'>
+            <a
+              href={`${STATIC_PATH}${submission.path}`}
+              target='_blank'
+            >
+              <Button color='primary'>Download</Button>
```

Comment:
Change this to 'view image' or something similar, if you're not auto downloading.